src/cls/IPM/General/SemanticVersionExpression/Comparator.cls

```diff
@@ -93,8 +93,9 @@ Method Evaluate(pVersion As %IPM.General.SemanticVersion) As %Boolean
       Set tEquals = tEquals && (val1 = val2)
     }
   }
-	If ($Extract(..Operator,1) = "=") {
-		// Means direct equality and not <= or >= so compare pre-release as well
+	If (##class(%IPM.Repo.UniversalSettings).GetValue("SemVerPostRelease") = 0) || ($Extract(..Operator,1) = "=")  {
```

Comment:
I would recommend coercing this to be a number with:
`If (+##class(%IPM.Repo.UniversalSettings).GetValue("SemVerPostRelease") = 0) || ($Extract(..Operator,1) = "=")  {`

That way null is 0 (the default)